lib/matplotlib/widgets.py

```diff
@@ -400,6 +429,8 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
         super().__init__(ax, orientation, closedmin, closedmax,
                          valmin, valmax, valfmt, dragging, valstep)
 
+        self.useblit = useblit and self.canvas.supports_blit
```

Comment:
Can this sort of thing live in `SliderBase`?

If blitting is going to be supported on slider then it ought to also be supported on `RangeSlider`